### PR TITLE
 ajout de sécurisation

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by Wowlfy — APPROVED** [review]

File: src/Entity/Job.php [hunk 1]
```diff
@@ -46,6 +46,13 @@ class Job
     /**
      * @ORM\Column(type="integer")
      * @Assert\NotBlank(message="Veuillez remplir ce champ s'il vous plait")
+     * @Assert\Length(max=2,
+     *     maxMessage = "Le nom de la ville ne peut pas dépasser {{ limit }} caractères")
+     * @Assert\Range(
+     *     min = 1,
+     *     max = 48,
+     *     notInRangeMessage = "vous devez choisir entre {{ min }}h et {{ max }}h.",
+     * )
```
Comment (LucyMerlier) [inline src/Entity/Job.php:55]: ?? I just don't understand those asserts

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by Erdnaxela45 — DISMISSED** [review]

File: src/Entity/Job.php [hunk 2]
```diff
@@ -46,6 +46,13 @@ class Job
     /**
      * @ORM\Column(type="integer")
      * @Assert\NotBlank(message="Veuillez remplir ce champ s'il vous plait")
+     * @Assert\Length(max=2,
+     *     maxMessage = "Le nombre d'heure ne peut pas dépasser {{ limit }} chiffres")
```
Comment (LucyMerlier) [inline src/Entity/Job.php:50]: this assert is nonsense => delete it

File: src/Entity/Job.php [hunk 3]
```diff
@@ -46,6 +46,13 @@ class Job
...
+     * @Assert\Range(
+     *     min = 1,
+     *     max = 48,
```
Comment (LucyMerlier) [inline src/Entity/Job.php:53]: 48 hours max? for a student job????

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by endricklosange — APPROVED** [review]

File: src/Entity/Job.php [hunk 4]
```diff
@@ -46,6 +46,11 @@ class Job
     /**
      * @ORM\Column(type="integer")
      * @Assert\NotBlank(message="Veuillez remplir ce champ s'il vous plait")
+     * @Assert\Range(
+     *     min = 1,
+     *     max = 35,
+     *     notInRangeMessage = "vous devez choisir entre {{ min }}h et {{ max }}h.",
```
Comment (LucyMerlier) [inline src/Entity/Job.php:52]: "Vous" => with capital "V"

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by endricklosange — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**